thingif/src/main/java/com/kii/thingif/KiiApp.java

```diff
@@ -50,7 +50,7 @@ public KiiApp(String appID, String appKey, Site site) {
      * @param hostName Host Name of the app.
      * @throws  IllegalArgumentException when appID, appKey and hostName is null or empty string.
      */
-    KiiApp(String appID, String appKey, String hostName) {
+    public KiiApp(String appID, String appKey, String hostName) {
```

Comment:
@shinsuke-nara The Builder is only for build KiiApp from hostname, which can be very complex(As document of KiiApp.Builder describes). So I think the Builder is necessary.